_posts/2020-04-12-final-presentations-youre-all-invited.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Graduation and final presentations - you're all invited!
+authors:
+- yochannah
+- malvikasharan
+- bebatut
+image: https://images.unsplash.com/photo-1560523159-6b681a1e1852?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2100&q=80
```

Comment:
```suggestion
image: https://images.unsplash.com/photo-1560523159-6b681a1e1852?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2100&q=80
photos:
  name: Product School
  url: https://unsplash.com/photos/ypZHyD1TqP0
```

Reply:
Then you may not need the last line

Reply:
oh, didn't know this was a thing - thanks!